.github/scripts/setup-env.sh

```diff
@@ -7,10 +7,11 @@ error() {
   echo "$1" >&2
 }
 
-while [[ $# -gt 0 ]]; do
+while [ $# -gt 0 ]; do
   case $1 in
     --compiler)
       COMPILER="$2"
+      break
```

Comment:
This will make this work better once we have more options
```suggestion
      shift
      shift
```